components/Dialogs/src/MDCAlertController.m

```diff
@@ -124,6 +124,15 @@ - (nonnull instancetype)initWithTitle:(nullable NSString *)title
   return self;
 }
 
+- (nonnull instancetype)initWithNibName:(nullable NSString *)nibNameOrNil
+                                 bundle:(nullable NSBundle *)nibBundleOrNil {
+  return [self initWithTitle:nil message:nil];
+}
+
+- (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder {
```

Comment:
Does initWithCoder still work with this change?

Reply:
We can make `initWithCoder:` available to superclasses, but the current design won't allow encoding of the full state of the AlertController. Specifically, the actions array is not possible to encode since its members (`MDCAlertAction *`) have block properties. If we were to switch to a delegate structure, it would be possible to encode the actions.